src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -3618,7 +3620,9 @@ namespace ZoneEquipmentManager {
                                        CoolingActive,
                                        OAUnitNum,
                                        OAUCoilOutTemp,
-                                       ZoneEquipFlag);
+                                       ZoneEquipFlag,
```

Comment:
call to zone equipment now includes SysOutputProvided and LatOutputProvided just like all other zone equipment